src/main/java/com/example/clinic/domain/Service.java

```diff
@@ -0,0 +1,81 @@
+package com.example.clinic.domain;
+
+
+public class Service {
+    private long id;
+    private String name;
+    private float fee;
+    private int percentInsurance;
+    private ThreadLocal startTime;
+    private ThreadLocal endTime;
```

Comment:
Lets simplify it to String where it can be "1h" or "1d" or "30m".

Reply:
simplified to String